project/npda/general_functions/csv_upload.py

```diff
@@ -57,15 +72,15 @@ def csv_upload(user, csv_file=None, organisation_ods_code=None, pdu_pz_code=None
     AuditCohort = apps.get_model("npda", "AuditCohort")
 
     # set previous quarter to inactive
-    AuditCohort.objects.filter(
+    await AuditCohort.objects.filter(
```

Comment:
since Django provides async versions of the ORM functionality I've simply used that where possible to avoid as much sync_to_async wrapping (since that implies a bigger code refactor pulling out functions all over the place)